### PR TITLE
upgrade klog@0.4.0 to klog/v2@2.2.0

### LINKED ISSUE #2346: empty log output when use release v1.5.0
**What happened**:
image
start cloudcore-v1.5.0, miss most basic logs.
**What you expected to happen**:
log well.

**Anything else we need to know?**:
I guess it is because we upgrade k8s to v.1.19.0, and it relay on k8s.io/klog/v2, but we still use k8s.io/klog to log as below:
image

**Environment**:
<details><summary>Kubernets Version</summary>

```console
$ kubelet --version
v1.5.0
$ kubectl version
# paste output here
```

</details>

<details><summary>CloudCore Version</summary>

```console
$ cloudcore --version
# paste output here
```

</details>

<details><summary>EdgeCore Version</summary>

```console
$ edgecore --version
# paste output here
```

</details>

**CloudSide Environment**:
<details><summary>Hardware configuration</summary>

```console
$ lscpu
# paste output here
```

</details>

<details><summary>OS</summary>

```console
$ cat /etc/os-release
# paste output here
```

</details>

<details><summary>Kernel</summary>

```console
$ uname -a
# paste output here
```

</details>

<details><summary>Go version</summary>

```console
$ go version
# paste output here
```

</details>

<details><summary>Others</summary>
</details>

**EdgeSide Environment**:
<details><summary>Hardware configuration</summary>

```console
$ lscpu
# paste output here
```

</details>

<details><summary>OS</summary>

```console
$ cat /etc/os-release
# paste output here
```

</details>

<details><summary>Kernel</summary>

```console
$ uname -a
# paste output here
```

</details>

<details><summary>Go version</summary>

```console
$ go version
# paste output here
```

</details>

<details><summary>Others</summary>
</details>

### PR DESCRIPTION
**Pull request by GsssC**
**What type of PR is this?**
upgrade klog to klog/v2

/kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #2346 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: hack/update-vendor-licenses.sh [hunk 1]
```diff
@@ -216,6 +217,13 @@ for PACKAGE in $(go list -m -json all | jq -r .Path | sort -f); do
     echo "${PACKAGE}, temporarily skipping" >&2
     continue
   fi
+  if [[ "${PACKAGE}" = "k8s.io/klog" ]]; then
```
Comment (GsssC) [inline hack/update-vendor-licenses.sh:220]: Temporarily add skip rules for work-around.
Upgrade this scripts later.

Comment (fisherxu) [reply hack/update-vendor-licenses.sh:220]: Vendor licenses can't pass?

Comment (GsssC) [reply hack/update-vendor-licenses.sh:220]: @fisherxu `update-vendor-licenses.sh` now regard path `k8s.io/klog` as a package, in fact it is not package but a path to "k8s.io/klog/v2" . It has been fixed by k8s master repo.

File: go.mod [hunk 2]
```diff
@@ -98,7 +98,7 @@ replace (
 	k8s.io/csi-translation-lib v0.0.0 => k8s.io/csi-translation-lib v0.0.0-20190718190424-bef8d46b95de
 	k8s.io/gengo v0.0.0 => k8s.io/gengo v0.0.0-20190327210449-e17681d19d3a // indirect
 	k8s.io/heapster => k8s.io/heapster v1.2.0-beta.1 // indirect
-	k8s.io/klog => k8s.io/klog v0.4.0 // indirect
+	k8s.io/klog => k8s.io/klog/v2 v2.2.0
```
Comment (fisherxu) [inline go.mod:101]: v2.4.0?

File: hack/update-vendor-licenses.sh (hunk 1 above)
Comment (fisherxu) [reply hack/update-vendor-licenses.sh:220]: >It has been fixed by k8s master repo.

Could we fix it in kubeedge?

Comment (fisherxu) [reply hack/update-vendor-licenses.sh:220]: > It has been fixed by k8s master repo.

Could we fix it in kubedge?

File: go.mod (hunk 2 above)
Comment (GsssC) [reply go.mod:101]: @fisherxu flow as k8s 1.19.3.
https://github.com/kubernetes/kubernetes/blob/1e11e4a2108024935ecfcb2912226cedeafd99df/go.mod#L477

File: hack/update-vendor-licenses.sh (hunk 1 above)
Comment (GsssC) [reply hack/update-vendor-licenses.sh:220]: @fisherxu There are some errors when directly upgrade this scriptto k8s master version. We could fix later in next pr.

Comment (GsssC) [reply hack/update-vendor-licenses.sh:220]: @fisherxu  recorded in #2350

File: go.mod (hunk 2 above)
Comment (fisherxu) [reply go.mod:101]: Others in this pr are v2.4.0..

Comment (GsssC) [reply go.mod:101]: @fisherxu `go mod tidy` auto upgrade require version to 2.4.0 when I set it to 2.2.0. Let me figure out why....

Comment (GsssC) [reply go.mod]: @fisherxu done, now both require and replace are v2.2.0, it is because I did not limited beehive and viaduct to v2.2.0 before.

**Comment by GsssC** [comment]
@fisherxu ready

File: go.mod (hunk 2 above)
Comment (fisherxu) [reply go.mod]: looks good

**Comment by fisherxu** [comment]
/lgtm
/approve
We need to cherry-pick it to 1.5 branch :) @GsssC

**Pull request merged**